architecture/hybrid-systems.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Hybrid systems
+summary: Systems deployed both in the cloud and on-premises
+reviewed: 2023-07-18
+---
+
+Hybrid systems have components running both in the cloud and on-premises. This might be a temporary situation in a lift and shift strategy, or it can be permanent state for various reasons (e.g. risk, legal, cost, etc.). Hybrid systems can become tricky to implement when the on-premises part of the system relies on an existing message broker. Some broker technologies might also not be compatible with modern cloud infrastructure (e.g. MSMQ).
+
+## Shared broker
```

Comment:
isn't that covered by the messaging bridge section which states:

> The Messaging Bridge pattern can be used to connect two separate messaging systems

Reply:
I guess that is sufficient

Reply:
thinking a bit more about it, maybe it would make more sense to have a new section on the same level as "shared broker" that is called "multiple brokers" where the messaging bridge is a solution-focused subsection of that section? Thoughts on that @andreasohlund ?

Reply:
I like it, that was kind of what I was after

Reply:
I tweaked the section (but didn't end up adding much more content after all), let me know if you feel there is something missing